docs/getting-started.md

```diff
@@ -21,7 +21,7 @@ def deps do
 end
 ```
 
-The current minimal version of elixir should be `1.3` and in this case you must add `phoenix_swagger` application
+The current minimal version of elixir should be `1.6` and in this case you must add `phoenix_swagger` application
```

Comment:
I think this sentence can be removed completely. The advice only applies to older elixir versions when Mix didn't start dependent applications automatically.